app/components/primer/beta/blankslate.rb

```diff
@@ -17,7 +17,11 @@ class Blankslate < Primer::Component
 
       # Optional graphic visual.
       #
-      # @param type [Symbol]  <%= one_of(Primer::Beta::Blankslate::GRAPHIC_OPTIONS) %>
+      # Use:
+      # - `graphic_icon` for an <%= link_to_component(Primer::OcticonComponent) %>.
```

Comment:
I think this is fine for now 👍🏻 We might want to look into something more automated in the future 😄